bundles/org.openhab.binding.miele/src/main/resources/OH-INF/thing/tumbledryer.xml

```diff
@@ -12,6 +12,7 @@
 
 		<label>Tumbledryer</label>
 		<description>This is a Miele@home compatible tumbledryer</description>
+		<semantic-equipment-tag>Dryer</semantic-equipment-tag>
```

Comment:
It might be worth considering adding `Tumble Dryer` as a synonym in https://github.com/openhab/openhab-core/blob/main/bundles/org.openhab.core.semantics/model/SemanticTags.csv